great_expectations/render/components.py

```diff
@@ -640,8 +643,8 @@ def __init__(
         self.params: Optional[dict] = params
 
         # TableType
-        self.header_row: Optional[List[RenderedAtomicValue]] = header_row
-        self.table: Optional[List[List[RenderedAtomicValue]]] = table
+        self.header_row: Optional[List[RendererTableValue]] = header_row
+        self.table: Optional[List[List[RendererTableValue]]] = table
```

Comment:
Would it make things easier if these defaulted to empty lists?
```suggestion
        self.header_row:List[RendererTableValue] = header_row or []
        self.table: List[List[RendererTableValue]] = table or []
```

Reply:
I tried this but the `None` indicates that the key should not be serialized. Otherwise we serialize the empty list.